src/Text/Pandoc/Readers/DocBook.hs

```diff
@@ -960,12 +960,22 @@ parseBlock (Elem e) =
                       let aligns = case colspecs of
                                      [] -> replicate numrows AlignDefault
                                      cs -> map toAlignment cs
+                      let parseWidth s = safeRead (T.filter (\x -> (x >= '0' && x <= '9')
+                                                                   || x == '.') s)
+                      let textWidth = case filterChild (named "?dbfo") e of
```

Comment:
What is this `?dbfo`?

Reply:
It's an XML processing instruction that's used by the docbook stylesheets to specify the table width in proportion to text. The CALS table model (from 1995) never included something width in relation to text - so this is what people came up with it seems.